.github/workflows/preview.yml

```
@@ -0,0 +1,240 @@
+  # ──────────────────────────────────────────────
+  cleanup:
+    if: github.event_name == 'pull_request' && github.event.action == 'closed'
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      pull-requests: write
+
+    steps:
+      - name: Compute branch slug
+        id: slug
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)